alpha/apps/kaltura/lib/multiLingualUtils.php

```diff
@@ -261,7 +261,7 @@ protected static function getMappedLanguages($newMultiLingualMapping, $supported
 			}
 		}
 		
-		return array_unique($mappedLanguages)[0];
+		return empty($mappedLanguages) ? null : array_unique($mappedLanguages, SORT_REGULAR)[0];
```

Comment:
since we expect array to return makes more sense to return array() rather than just null if the $mappedLanguages array is empty